buku

```diff
@@ -618,11 +619,15 @@ class BukuDb:
             return
 
         if not self.json:
-            prompt(results, self.noninteractive)
+            if delete is False:
+                prompt_open(results, self.noninteractive)
```

Comment:
My only concern is the consistency of the cmd line flags.
From the user point of view, he can delete individual records and even ranges by using "-d" flag, then if he wants to delete using different criteria he has to use only "-s" which would also allow deletion.. :)